timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -28,6 +28,10 @@ types:
       base-type: any
       external:
         java: java.lang.Long
+    LockWatchReference:
+      base-type: any
+      external:
+        java: com.palantir.lock.watch.LockWatchReferences.LockWatchReference
```

Comment:
not particularly fussed but not sure where we stand re implementing the union in conjure?

Reply:
Yeah, would be nice, but LWReferences are already included in `LockWatchStateUpdate`, so that change would break the in-use start transaction endpoint 🐛